Basic-Car-Maintenance/Shared/Settings/Views/AddVehicleView.swift

```diff
@@ -14,6 +14,10 @@ struct AddVehicleView: View {
     @State private var name = ""
     @State private var make = ""
     @State private var model = ""
+    @State private var year = ""
+    @State private var color = ""
+    @State private var VIN = ""
```

Comment:
While this does make sense to capitalize, because it's an abbreviation, instead though I want to make it all lowercase